spec/std/math/stats_spec.cr

```diff
@@ -0,0 +1,8 @@
+require "spec"
+
+describe "Stats" do
+  it "mean" do
+    Math.mean([1, 3.5, 9]).should eq(4.5)
+    Math.mean({1, 3.5, 9}).should eq(4.5)
```

Comment:
These are two different ways of calculating the mean, maybe having two different tests would be better, don't you think? 

It also would serve the purpose of documenting the code. In the likes of:

```crystal
it "calculates the mean of an Array" do
  Math.mean([1, 3.5, 9]).should eq(4.5)
end

it "calculates the mean of a Tuple" do
  Math.mean({1, 3.5, 9}).should eq(4.5)
end
```